src/components/cuda/cupti_profiler.c

```diff
@@ -2482,20 +2694,30 @@ static int evt_code_to_name(uint64_t event_code, char *name, int len)
   *   Structure for member variables such as symbol, short description, and 
   *   long desctiption. 
 */
-int cuptip_evt_code_to_info(uint64_t event_code, PAPI_event_info_t *info)
+int cuptip_evt_code_to_info(uint32_t event_code, PAPI_event_info_t *info)
 {
-    int papi_errno, i, gpu_id;
-    char description[PAPI_HUGE_STR_LEN];
-
-    /* get the events nameid and flags */
+    int papi_errno, i;
     event_info_t inf;
+    char all_stat[PAPI_HUGE_STR_LEN]="", basename[PAPI_HUGE_STR_LEN]="";
+
     papi_errno = evt_id_to_info(event_code, &inf);
     if (papi_errno != PAPI_OK) {
         return papi_errno;
     }
+    
+    const char *stat_position = strstr(cuptiu_table_p->events[inf.nameid].basename, "stat");
+    if (stat_position == NULL) {
+        return PAPI_ENOMEM;
+    }
+    size_t prefix_len = stat_position - cuptiu_table_p->events[inf.nameid].basename;
```

Comment:
Same as the previous mention of `prefix_len` I would suggest changing this to `basename_len`.